src/components/classifier/QuestionClassifier.js

```diff
@@ -0,0 +1,323 @@
+                >
+                  <View style={styles.backgroundView} />
+                  <View style={styles.classifierContainer}>
+                    <View onLayout={({nativeEvent}) => this.setState({imageDimensions: {width: nativeEvent.layout.width, height: nativeEvent.layout.height}})}>
+                      <TapableSubject
+                        height={300}
+                        width={this.state.imageDimensions.width}
+                        subject={subject}
+                        alreadySeen={seenThisSession}
+                        onPress={(imageSource) => this.setState({
+                          showFullSize: true,
+                          fullScreenImageSource: {uri: imageSource}})}
+                      />
+                    </View>
+                    {
+                      answers.map((answer, index) =>
+                        <View key={index} style={styles.buttonContainer}>
+                          <ClassifierButton
+                            selected={index === this.state.answerSelected}
+                            blurred={this.state.answerSelected !== -1 && index !== this.state.answerSelected}
+                            type="answer"
+                            text={answer.label}
+                            onPress={this.onOptionSelected(index)}
+                          />
+                        </View>
+                      )
+                    }
+                  </View>
+                  <View style={styles.buttonContainer}>
+                    <ClassifierButton
+                      disabled={answerSelected === -1}
+                      type="answer"
+                      text="Submit"
+                      onPress={this.submitClassification()}
+                    />
+                  </View>
+                  {
+                    (task.help || R.length(guide.items) > 0) && 
+                      <Separator style={styles.separator}/>
+                  }
+                  { 
+                    task.help && 
+                      <NeedHelpButton 
+                        onPress={() => this.classifierContainer.displayHelpModal()}
+                      />
+                  }
+                  {
+                    R.length(guide.items) > 0 &&
+                      <ClassifierButton
+                        onPress={() => this.classifierContainer.displayFieldGuide()}
+                        style={styles.guideButton}
+                        text="Field Guide"
+                        type="guide"
+                      />
+                  }
+                </ScrollView>
+              :
+                renderTutorial()
+            }
+            <FullScreenImage
+              source={this.state.fullScreenImageSource}
+              isVisible={this.state.showFullSize}
+              handlePress={() => this.setState({ fullScreenQuestion: '', showFullSize: false })}
+              question={this.state.fullScreenQuestion}
```

Comment:
Can you destructure these state items as you've done with several other items at the beginning of the `render()` function?